deployment/docker-compose/docker-compose.yml

```diff
@@ -71,20 +71,23 @@ services:
       /usr/local/bin/python server.py --port=5001 --models=base
     restart: always
   embeddings:
-    image: idinsight/embeddings:latest
+    image: ${EMBEDDINGS_IMAGE:-ghcr.io/huggingface/text-embeddings-inference:cpu-1.5}
```

Comment:
This is to have the option to use local image compatible for  arm device when deploying locally and using the huggingface image when deploying online

Reply:
what part of this code determines this?

Reply:
ohh the "EMBEDDINGS_IMAGE"... Wow this is a complex process. Can you document the full process on a page in mkdocs? I think "using local embeddings" might need its own page to explain everything.

Reply:
Added ticket to the backlog